governance/xc_admin/packages/xc_admin_common/src/multisig_transaction/BpfUpgradableLoaderMultisigInstruction.ts

```diff
@@ -62,19 +62,32 @@ export class BpfUpgradableLoaderInstruction implements MultisigInstruction {
             }
           );
         case 5:
-          return new BpfUpgradableLoaderInstruction(
-            "Close",
-            {},
-            {
+          let args;
+          // Close instruction supports closing two types of accounts:
+          // - A program which takes 4 keys (programData, spill, upgradeAuthority, program)
+          // - A buffer which takes 3 keys (buffer, spill, upgradeAuthority)
+          if (instruction.keys.length == 4) {
```

Comment:
IMO better `(instruction.keys.length >= 4)` to handle the other case

Reply:
makes sense.